src/client/queue.ts

```diff
@@ -0,0 +1,128 @@
+import type { PublishRequest, PublishResponse } from "./client";
+import type { Requester } from "./http";
+import { prefixHeaders, processHeaders } from "./utils";
+
+export type TQueue = {

```

Comment:
Let's rename `TQueue` to `QueueResponse` or `QueueResult`. `T` prefix generally stands for generics like `TResult`, `TResponse`, `TMetadata` etc...

Reply:
Ah ok